src/components/ColorPicker/ColorPicker.scss

```diff
@@ -99,7 +105,21 @@ $huepicker-bottom-padding-start: $picker-size - $dragger-size;
   width: rem(24px);
   margin-left: spacing(tight);
   border-width: var(--p-border-radius-base);
-  border-radius: $picker-size * 0.5;
+  border-radius: $vertical-picker-border-radius;
+
+  &::after {
```

Comment:
This looks great! 

I wonder if there's value in also adding a High Contrast Mode outline to the respective containers. 

Windows HCM tophat:

![Screenshot 2022-01-25 at 13 51 12](https://user-images.githubusercontent.com/17032120/150989832-966b4818-e128-4d03-9000-cd92fa22d7f9.png)

In the assembled GradientPicker, the relevant info will be conveyed numerically as well as visually (I think?), so the lack of gradient isn't an issue, but adding an HCM outline to the containing elements might make the overall experience a little easier to understand for HCM users. Just a though though, and not a blocker for this PR!